docs/docsite/rst/reference_appendices/common_return_values.rst

```diff
@@ -20,55 +20,202 @@ backup_file
 ```````````
 For those modules that implement `backup=no|yes` when manipulating files, a path to the backup file created.
 
+    .. code-block:: console
+
+      "backup_file": "./foo.txt.32729.2020-07-30@06:24:19~"
+    
+
 changed
 ```````
 A boolean indicating if the task had to make changes.
 
+    .. code-block:: console
+
+        "changed": true
+
 diff
 ````
 
+    .. code-block:: console
+
+        "diff": [
+                {
+                    "after": "",
+                    "after_header": "foo.txt (content)",
+                    "before": "",
+                    "before_header": "foo.txt (content)"
+                },
+                {
+                    "after_header": "foo.txt (file attributes)",
+                    "before_header": "foo.txt (file attributes)"
+                }
+
 Information on differences between the previous and current state. Often a dictionary with entries ``before`` and ``after``, which will then be formatted by the callback plugin to a diff view.
```

Comment:
I'd put this sentence above the sample codeblock, so all the entries follow the same pattern (Title, description, example).

Reply:
Thanks @acozine!